Readme.md

```diff
@@ -2,8 +2,13 @@
 - Install nodejs
```

Comment:
No need in line 1. Node should be installed for using JS, but not your framework. I mean Node is a must have thing for work.